src/core/backend_context.cc

```diff
@@ -270,7 +272,9 @@ BackendResponder::Finalize()
     char* pinned_buffer = def.pinned_memory_->MutableBuffer(
         &pinned_memory_type, &pinned_memory_id);
 
-    size_t offset = 0;
+    SyncQueue<std::pair<Status, bool>> completion_queue;
+    std::vector<size_t> async_task_ids;
```

Comment:
I don't understand what async_task_ids is for. "count" should be named "async_copy_count" and I don't see why that isn't enough.

Reply:
The count was for all counts and not just async_copy counts. This will be removed when response is sent as a param to the task